Cargo.toml

```diff
@@ -56,7 +56,7 @@ protobuf-codegen-pure = "~2.10"
 all-features = true
 
 [features]
-blocking = ["tokio/rt-threaded"]
+blocking = []
```

Comment:
Removed this because we changed it to be always enabled at the dependency level (line 41)